### PR TITLE
Settings Dialog ->Settings Panel

### LINKED ISSUE #751: Consider showing the current Aspire version in the dashboard
We could put it in a footer or in the settings dialog. Would be nice to be able to immediately identify what version is running.

### PR DESCRIPTION
**Pull request by vnbaaij**
Using a different method call to turn the Settings dialog to a Settings panel (docked to the right side). By doing this room is created to also display the Dashboard version number (fix #751):
image

### CODE REVIEW

File: src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor [hunk 1]
```diff
@@ -2,8 +2,13 @@
 @inject IJSRuntime JS
 @implements IDialogContentComponent
 
-<FluentRadioGroup TValue="string" Label="Theme" Value="@_currentSetting" ValueChanged="SettingChangedAsync" Orientation="Orientation.Vertical">
-    <FluentRadio Value="@ThemeSettingSystem">System</FluentRadio>
-    <FluentRadio Value="@ThemeSettingLight">Light</FluentRadio>
-    <FluentRadio Value="@ThemeSettingDark">Dark</FluentRadio>
-</FluentRadioGroup>
+<FluentStack Orientation="Orientation.Vertical" Style="display: contents
+">
```
Comment (tlmii) [inline src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor:6]: errant line break here within the Style attribute?

File: src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor [hunk 2]
```diff
@@ -2,8 +2,13 @@
...
+    <FluentRadioGroup TValue="string" Label="Theme" Value="@_currentSetting" ValueChanged="SettingChangedAsync" Orientation="Orientation.Vertical">
+        <FluentRadio Value="@ThemeSettingSystem">System</FluentRadio>
+        <FluentRadio Value="@ThemeSettingLight">Light</FluentRadio>
+        <FluentRadio Value="@ThemeSettingDark">Dark</FluentRadio>
+    </FluentRadioGroup>
+    <div style="height: 100%;"></div>
```
Comment (tlmii) [inline src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor:12]: Seems a little weird to have the extra div here just to take up the space. Could this alignment be done by setting margin-top: auto; on the version div below instead? It's not that big of a deal. Just curious

Comment (vnbaaij) [reply src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor:12]: I couldn't get it to line up nicely with margin-top. Sometimes with CSS I choose the path of least resistance...

File: src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor (hunk 1 above)
Comment (vnbaaij) [reply src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor:6]: fixed

**Review by tlmii — APPROVED** [review]

**Pull request merged**